playwright/e2e/timeline/timeline.spec.ts

```diff
@@ -916,7 +918,26 @@ test.describe("Timeline", () => {
             await page.getByRole("button", { name: "Hide" }).click();
 
             // Check that the image is now hidden.
-            await expect(page.getByRole("link", { name: "Show image" })).toBeVisible();
+            await expect(page.getByRole("button", { name: "Show image" })).toBeVisible();
+        });
+
+        test("should be able to hide a video", { tag: "@screenshot" }, async ({ page, app, room, context }) => {
```

Comment:
I don't think you need the screenshot tag here if you don't assert a screenshot.